ceph/rados/mgr_workflows.py

```diff
@@ -0,0 +1,494 @@
+                host=hostname, daemon_type="mgr"
+            ):
+                log.info(f"Mgr {hostname} is add to the cluster.")
+                return True
+            log.info(
+                "Mgr daemon still not added.sleeping for 5 seconds and checking again"
+            )
+            time.sleep(5)
+        return False
+
+    def remove_mgr_service(self, hostname) -> bool:
+        """
+        Method to remove the mgr service from the given host
+
+        Args:
+            hostname: name of the host from where mgr needs to be removed
+
+        returns:
+            Pass -> True, Fail -> false
+        """
+
+        if not self.rados_obj.check_daemon_exists_on_host(
+            host=hostname, daemon_type="mgr"
+        ):
+            log.info(f"Provided {hostname} mgr is not present in the  mgr list.")
+            return False
+        # Get the current mgr list
+        current_mgrs = self.get_mgr_list()
+        current_mgrs.remove(hostname)
+        no_mgrs = len(current_mgrs)
+        new_mgr_list = " ".join(current_mgrs)
+        cmd = f'ceph orch apply mgr "{no_mgrs} {new_mgr_list}" '
+        try:
+            self.client.exec_command(sudo=True, cmd=cmd)
+            time.sleep(120)
+        except Exception as err:
+            log.debug(f"Hit issue during command execution : {err}")
+
+        mgr_exists = True
+        endtime = datetime.datetime.now() + datetime.timedelta(seconds=180)
+        while datetime.datetime.now() < endtime:
+            time.sleep(10)
+            if not self.rados_obj.check_daemon_exists_on_host(
+                host=hostname, daemon_type="mgr"
+            ):
+                log.info(f"Mgr daemon is removed on the host {hostname}.")
+                mgr_exists = False
+                break
+            log.info(
+                f"Mgr daemon still present on the host {hostname}. sleeping for 5 seconds and checking again"
+            )
+        if mgr_exists:
+            log.error(
+                f"Mgr daemon still present on the host {hostname}. Failed to remove"
+            )
+            return False
+        log.debug(f"Mgr successfully removed from host : {hostname}")
+        return True
+
+    def remove_mgr_with_label(self, hostname):
+        """
+        Method to remove the mgr service from the given host
+        Args:
+            hostname:  Cluster hostname  where mgr needs to be remove
+        returns:
+                Pass -> True, Fail -> false
+        """
+        # Checking if mgr service exists before removing
+        if not self.rados_obj.check_daemon_exists_on_host(
+            host=hostname, daemon_type="mgr"
+        ):
+            log.info(f"Provided {hostname} mgr is not present in the  mgr list.")
+            return False
+
+        cmd = f"ceph orch host label rm {hostname} mgr"
+        self.client.exec_command(sudo=True, cmd=cmd)
+        time.sleep(120)
+        log.debug(f"Added mgr label for the host : {hostname}")
```

Comment:
I believe this should say Removed mgr label for the host..........

Reply:
Updated the string